src/calculateRentalCost.js

```diff
@@ -28,7 +28,24 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rent = 40;
+  let cashBack = 0;
```

Comment:
I would better divide this one into two different discount variables (declared with const)